lib/galaxy/webapps/tool_shed/buildapp.py

```diff
@@ -80,7 +80,7 @@ def app_factory( global_conf, **kwargs ):
                       image_file=None )
     webapp.add_route( '/{controller}/{action}', action='index' )
     webapp.add_route( '/{action}', controller='repository', action='index' )
-    webapp.add_route( '/repos/*path_info', controller='hg', action='handle_request', path_info='/' )
+    webapp.add_route( '/repos/{owner}/{name}/*path_info', controller='hg', action='handle_request', path_info='/' )
```

Comment:
what can `path_info` be here? it seems that it was always a combination of `/owner/name/`.

This might also change behaviour as previously if you called this without `name` it would give you `wsgi_app` but now it will give you missing param.